core/embed/rust/src/ui/model_tt/component/frame.rs

```diff
@@ -187,3 +185,28 @@ where
         t.close();
     }
 }
+
+pub fn icon_text_center(
```

Comment:
this function seems misplaced here, probable would be better in display::mod?

Reply:
it is a better choice since it can be reused outside of TT, we can figure out something else if we gather more of these helpers or `ui::util` bloats too much.